packages/cli/src/modules/insights/insights.config.ts

```diff
@@ -43,4 +43,18 @@ export class InsightsConfig {
 	 */
 	@Env('N8N_INSIGHTS_FLUSH_INTERVAL_SECONDS')
 	flushIntervalSeconds: number = 30;
+
+	/**
+	 * The maximum age in days for all insights data before pruning.
+	 * Default: -1 (no pruning)
+	 */
+	@Env('N8N_INSIGHTS_MAX_AGE_DAYS')
```

Comment:
Most env vars don't include the unit, but env vars in this module do include it. Not sure what's the convention.